### PR TITLE
2.4 i18n - post merge string fixes

### PR DESCRIPTION
**Pull request by esbrandt**
Minor fixes for errors found while translating strings, post merge https://github.com/mixxxdj/mixxx/pull/3728

### CODE REVIEW

File: src/encoder/encoderfdkaac.cpp [hunk 1]
```diff
@@ -261,10 +261,10 @@ int EncoderFdkAac::initEncoder(int samplerate, QString* pUserErrorMessage) {
             // TODO(Palakis): write installation guide on Mixxx's wiki
             // and include link in message below
             *pUserErrorMessage = QObject::tr(
-                    "<html>Mixxx cannot record or stream in AAC "
+                    "Mixxx cannot record or stream in AAC "
```
Comment (Holzhaus) [inline src/encoder/encoderfdkaac.cpp:264]: Are you sure this isn't to force rich formatting?

Comment (uklotzde) [reply src/encoder/encoderfdkaac.cpp:264]: This also applies to 2.3. If we fix it we should fix it there first.

**Comment by uklotzde** [comment]
The first commit belongs into main and should be merged asap. The second commit should be reverted and published as a separate PR.

**Comment by esbrandt** [comment]
> The first commit belongs into main and should be merged asap. The second commit should be reverted and published as a separate PR.

Done

**Review by uklotzde — APPROVED** [review]
LGTM (waiting for CI)

**Comment by uklotzde** [comment]
No need to wait for the build-checks.

**Pull request merged**